nrf_security/Kconfig

```diff
@@ -689,23 +689,29 @@ config GLUE_MBEDTLS_DHM_C
 
 config CC310_GLUE_MBEDTLS_DHM_C
 	bool
+	depends on CC310_MBEDTLS_DHM_C && VANILLA_MBEDTLS_DHM_C
+	default y
 	select GLUE_MBEDTLS_DHM_C
 
 config VANILLA_GLUE_MBEDTLS_DHM_C
 	bool
+	depends on CC310_MBEDTLS_DHM_C && VANILLA_MBEDTLS_DHM_C
+	default y
 	select GLUE_MBEDTLS_DHM_C
 
 config CC310_MBEDTLS_DHM_C
-	bool "cc310" if CC310_BACKEND
-	default y
+	bool "cc310"
+	depends on CC310_BACKEND
+	default y if CC310_BACKEND
```

Comment:
I think the condition here is redundant to the depends on statement

Reply:
BTW. this does not change anything so no need to fix it if this is going to be the only change.

Reply:
> I think the condition here is redundant to the depends on statement

correct.